From c04b34672b59c8d1b35aaaa61cba4544e1068d9e Mon Sep 17 00:00:00 2001
From: "William A. Kennington III" <william@wkennington.com>
Date: Sat, 29 Dec 2018 11:20:09 -0800
Subject: [PATCH 3/4] timezone: Fix zoneinfo path for triton

This is just needed for the zic(8) tool so we don't need to worry about
supporting normal distros which use /usr/share/zoneinfo since they will
have their own zic(8) or can specify a path manually.
---
 timezone/tzfile.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/timezone/tzfile.h b/timezone/tzfile.h
index 27a38cc74d..b7e522e277 100644
--- a/timezone/tzfile.h
+++ b/timezone/tzfile.h
@@ -22,7 +22,7 @@
 */
 
 #ifndef TZDIR
-#define TZDIR	"/usr/share/zoneinfo" /* Time zone object file directory */
+#define TZDIR	"/run/current-system/sw/share/zoneinfo" /* Time zone object file directory */
 #endif /* !defined TZDIR */
 
 #ifndef TZDEFAULT
-- 
2.20.1.495.gaa96b0ce6b-goog

